.github/workflows/release_jars.yml

```diff
@@ -0,0 +1,48 @@
+# This action will publish jars that can be used by third parties.
+# Pretty much everything is done through gradle.
+# The version used will be according to the axion-release-plugin, meaning it will take a tag if present,
+# if not it will create a SNAPSHOT version.
+# gradle will compile, sign then upload artifacts to a Sonatype staging repo.
+# See https://s01.oss.sonatype.org for accessing these repos.
+# At this point it should manually be closed, which will trigger acceptance tests for maven central (but not transfer yet)
+# Once closed, thew repo is available for testing.
+# After testing, it can be manually promoted on the sonatype site, which will then publish to maven central.
+name: Upload Release Jars to Sonatype
+
+on:
+  release:
+    types: [ prereleased, released ]
+    workflow_dispatch:
```

Comment:
It might be more generic to trigger the action by listening to tag pushes so that any new tag will be published. 
Example:
```
on:
  push:
    tags:
      - '*'
```

Reply:
Sure. But it should be restricted to tags that have the proper syntax (e.g. v4.2.1), If not the axion-release plugin will not work. I initially was using an uppercase V at the beginning of the version, and it did not work! It came with some headaches.

Reply:
Did not put tag as a trigger in the end since the other part of publish_assets.yml needs a release trigger.